plan/statistics/statistics.go

```diff
@@ -37,7 +37,7 @@ const (
 	// between condition selects 1/40 of total rows.
 	pseudoRowCount    = 10000000
 	pseudoEqualRate   = 1000
-	pseudoLessRate    = 3
+	pseudoLessRate    = 30
```

Comment:
This rate number indicates a less condition has 1/30 possibility to pass, which does not reflect the fact.